### PR TITLE
Recreate Flutter project with AndroidX dependencies.

### PR DESCRIPTION
**Pull request by sixbangs**
Also removes some ignored files.

### CODE REVIEW

File: ios/Classes/GeofencingPlugin.m [hunk 1]
```diff
@@ -1,7 +1,3 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
```
Comment (bkonyi) [inline ios/Classes/GeofencingPlugin.m]: Please leave the copyright as-is.

File: lib/geofencing.dart [hunk 2]
```diff
@@ -2,8 +2,164 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-export 'package:geofencing/src/geofencing.dart'
-    hide geofenceEventToInt, intToGeofenceEvent;
```
Comment (bkonyi) [inline lib/geofencing.dart]: Is there any reason this was removed and the other source files moved? The purpose of this file is to hide internal methods.

**Review by bkonyi — CHANGES_REQUESTED** [review]
Thanks for updating! I've been meaning to do this for awhile.

File: ios/Classes/GeofencingPlugin.m (hunk 1 above)
Comment (sixbangs) [reply ios/Classes/GeofencingPlugin.m]: I thought it was included by accident; will fix.

File: lib/geofencing.dart (hunk 2 above)
Comment (sixbangs) [reply lib/geofencing.dart]: Fixed. I had missed the directory structure here.

**Review by bkonyi — APPROVED** [review]

**Comment by bkonyi** [comment]
Sorry for the delay, thanks for the fix!

**Comment by desarrollo-cam10** [comment]
Hi,
This update isn't in flutter package "geofencing: 0.0.1". Do you have to upload?
Thanks in advance

**Comment by jesusmartinezd** [comment]
Hi,

Is this merge upload to flutter package? I get same error than previous (geofencing: "0.0.1").

Thanks in advance

**Pull request merged**